src/components/AnnualReport/AGMDate.vue

```diff
@@ -112,193 +112,148 @@
   </v-card>
 </template>
 
-<script lang="ts">
-import Vue from 'vue'
-import { Component, Mixins, Prop, Watch, Emit } from 'vue-property-decorator'
-import { State, Getter } from 'vuex-class'
-import { DateMixin } from '@/mixins'
+<script setup lang="ts">
+import { reactive } from 'vue'
+import { AllowableActionsComposable, DateComposable } from '@/composables'
 import { FormIF } from '@/interfaces'
-
-@Component({})
-export default class AgmDate extends Mixins(DateMixin) {
```

Comment:
The composition API allows hacky versions of implementing inheritance functionality, initially switching to the composition api will lead to excess code   ```@Getter getCurrentDate!: string``` becomes ```const { getCurrentDate } = DateComposable``` or  ```const { getCurrentDate } = FilingComposable()``` Both work and lead to code having to be written in in DateComposable and FilingComposable which could be refactored to reduce redundant code after the inital switch

Reply:
Can we use a mix of Composition API and Options API?

Can we use a mix of Composables and Mixins?

Reply:
From what I understand the options api uses the composition api under the hood, so yes you can use both together. However that can lead to issues initially because all of the composables and components  will be initialized first. Even before the beforeCreate() in the options api.

Reply:
OK, thanks.

What about using Composables and Mixins at the same time... possible?

Reply:
I don't see any reason why you couldn't.